cmd/veto/update_runtime.go

```
@@ -0,0 +1,306 @@
+	}
+	for _, asset := range release.Assets {
+		delete(requiredAssets, asset.Name)
+	}
+	if len(requiredAssets) != 0 {
+		return "", errors.New("latest release artifact set is incomplete")
+	}
+	return version, nil
+}
+
+func newUpdateHTTPClient() *http.Client {
+	transport := http.DefaultTransport.(*http.Transport).Clone()
```

golangci-lint (2.12.2)

[error] right hand must be only type assertion

(forcetypeassert)